src/simulation/LoadGenerator.cpp

```diff
@@ -382,23 +425,17 @@ LoadGenerator::submitPaymentOrPretendTx(GeneratedLoadConfig const& cfg,
             status != TransactionQueue::AddResult::ADD_STATUS_ERROR)
         {
             mFailed = true;
-            return 0;
+            return false;
```

Comment:
you missed the `return cfg.nTxs;` above that should be `false` I think given the refactor (C++ is nice like that)

Reply:
Nice catch, fixed.